Sources/SuperValidator/Validators/Phone.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  File.swift
```

Comment:
Can you rename this file to PhoneNumber, and all prefix use PhoneNumber
e.g PhoneNumberFormat, PhoneNumberError, etc.

Reply:
got it